lib/server.ts

```diff
@@ -15,6 +15,7 @@ import { readFile, readJson, promises } from "./utils/fs";
 export const DEFAULT_LOCAL_GATEWAY_PATH = path.join(__dirname, "../..", "gateway", "dist");
 
 export const DEFAULT_REMOTE_GATEWAY_URL = "https://ipfs.web4.near.page/ipfs/bafybeibe63hqugbqr4writdxgezgl5swgujay6t5uptw2px7q63r7crk2q/";
```

Comment:
I think there is a cleaner solution with just DEFAULT_GATEWAY object with these as default values, instead of having to maintain all these exports + constants